features/paint-order.yml

```diff
@@ -0,0 +1,7 @@
+name: paint-order
+description: "The `paint-order` CSS property sets whether the `stroke` is above or below the `fill`. In SVG content, `markers` can also be ordered. For example, `paint-order: markers stroke fill;` will draw the markers, then stroke, then fill."
+spec: https://svgwg.org/svg2-draft/painting.html#PaintOrder
+group: css
+compat_features:
+  - css.properties.paint-order
+  - svg.global_attributes.paint-order
```

Comment:
I probably ought to have asked for this previously, but there's no time like the present: are SVG attributes _part_ of their corresponding CSS properties? I'm not sure that's quite right, but it's not _wrong_ either. I think there's two ways to handle this:

- Keep including SVG attrs, but also start assigning these things to an SVG group (maybe even an SVG-in-CSS group?).
- Drop the SVG attributes, then consider SVG more holistically later on.

Reply:
I think it's pretty helpful for authors to associate the SVG and CSS versions of a feature like this. They really are the same feature - similar to JS API's being included along side CSS. But I expect in most cases SVG support will be significantly broader/farther back, since these generally start in SVG and then get adopted into CSS. Ideally, if they're not together, they are at least clearly associated.

Reply:
OK, cool. This works for me. I've had a bit of a look around and it does seem common for SVG and CSS to mentioned in the same setting—let's keep these together. If we want to group the SVG stuff, we can do that later.